src/k8s-job-exit-spec/config/k8s-job-exit-spec.yaml

```diff
@@ -780,6 +799,21 @@ spec:
   solution:
   - "Contact PAI Dev to fix the PAI Runtime bug"
 
+- code: 255
+  phrase: UnrecognizedUserErrorReceived
+  issuer: USER_CONTAINER
+  causer: USER_CONTAINER
+  type: USER_FAILURE
+  stage: RUNNING
+  behavior: PERMANENT
+  reaction: NEVER_RETRY
+  reason: "User command return with unrecognized exit pattern"
+  repro:
+  - "Retry the job with same command"
+  solution:
+  - "Check job logs"
+  - "Correct the command and retry the job"
+
```

Comment:
This should be unknown, pls refer previous spec on yarn :

- code: 255
  phrase: CONTAINER_RUNTIME_UNKNOWN_FAILURE
  issuer: PAI_RUNTIME
  causer: UNKNOWN
  type: UNKNOWN_FAILURE
  stage: COMPLETING
  behavior: UNKNOWN
  reaction: RETRY_TO_MAX
  reason: "Container failed but the failure cannot be recognized by PAI Runtime"
  repro:
    - "User program directly exits with exitcode 1"
  solution:
    - "Check container log and find root cause"
    - "Wait result from next retry"